pycode/memilio-simulation/memilio/simulation/compartments/compartmentalmodel.h

```diff
@@ -1,22 +1,43 @@
+/* 
+* Copyright (C) 2020-2021 German Aerospace Center (DLR-SC)
+*
+* Authors: Martin Siggel, Daniel Abele, Martin J. Kuehn, Jan Kleinert, Maximilian Betz
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
 #ifndef PYMIO_COMPARTMENTALMODEL_H
 #define PYMIO_COMPARTMENTALMODEL_H
 
 #include "memilio/compartments/compartmentalmodel.h"
 
 #include "pybind11/pybind11.h"
 
+namespace py = pybind11;
```

Comment:
There should not be `using namespace ...` or alias namespaces `namespace x = y` in a header, since this changes the namespace for every file that includes the header, which could lead to name conflicts that are hard to diagnose and/or solve. In headers, always use the full namespace. In cpp files, namespace aliases or usings can be ok.